src/index.css

```diff
@@ -1,5 +1,5 @@
 body {
-  background: #F4F4F4;
+  background: #1B2631;
```

Comment:
Oon vähän hämmentynyt kun tajusin että meillä on täällä näitä sass:sta generoitavia css-tiedostoja